src/api/pokemons.js

```diff
@@ -0,0 +1,16 @@
+import { format } from "prettier"
+import { App } from "../app/App"
+
+const POKEMON_API_BASE_URL = process.env.POKEMON_API_BASE_URL || "https://pokeapi.co/api/v2/";
+
+export async function getPokemonById(id) {
+    const res = await fetch(`${POKEMON_API_BASE_URL}pokemon/${id}`);
```

Comment:
It looks like using fetch with jest is not out of the box, you need to import:
`import fetch from 'cross-fetch';`
according to: [](https://stackoverflow.com/questions/48433783/referenceerror-fetch-is-not-defined/51466112)

Reply:
this library is already in our package.json, so don't worry about npm